scripts/ci/docker-compose/base.yml

```diff
@@ -59,6 +59,8 @@ services:
       - DB_RESET
       - ENABLED_SYSTEMS
       - RUN_SYSTEM_TESTS
+      - DEFAULT_PYTHON_SHORT_VERSION
+      - PYTHON_SHORT_VERSION
```

Comment:
Do we need both?

Reply:
Not any more :). It was only needed to generate requirements.txt file in the main repo but I removed it - it was nice for IDE's but very redundant and confusing.